Content.Shared/Preferences/SpawnPriorityPreference.cs

```diff
@@ -0,0 +1,14 @@
+namespace Content.Shared.Preferences
+{
+    /// <summary>
+    /// The spawn priority preference for a profile. Stored in database!
+    /// </summary>
+    public enum SpawnPriorityPreference
+    {
+        ///////////////////////
+        /// DO NOT TOUCH!!! ///
+        ///////////////////////
+        Arrivals = 0,
+        Cryosleep = 1
+    }
```

Comment:
You should make 0 no preference as the default, then 1 arrivals 2 cryosleep I think.